dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -673,7 +673,11 @@ html, body {
   text-align: center;
 }
 
+
 .header_button {
+  &.header_button_light {
+    background-color: #80d6de;
```

Comment:
is this one of the standard colors? if so should we use the name? if not should we add a special case name?